commit f7b3ec179f26187c5e8dfa4f56d7a31b749aad04 (refs/changes/27/3227/2)
Author: Angela Fong <angela.fong@joyent.com>
Date:   2018-01-17T13:15:23-08:00 (1 year, 9 months ago)
    
    ADMINUI-2396 Roll back commit 9f6a5ea (for ADMINUI-2367)
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/lib/ufds.js b/lib/ufds.js
index 62f2a754..9c20ba4b 100644
--- a/lib/ufds.js
+++ b/lib/ufds.js
@@ -39,15 +39,10 @@ module.exports = {
 
         ufds.on('timeout', function (msg) {
             log.warn({message: msg}, 'UFDS client timeout (recycling)');
+            ufds.close();
         });
 
-        var connectTimout = setTimeout(function () {
-            log.error('Connecting to UFDS failed ', config.url);
-            reconnect();
-        }, config.connectTimeout);
-
         ufds.once('connect', function () {
-            clearTimeout(connectTimout);
             log.info('Connected to UFDS: ', config.url);
             ufds.removeAllListeners('error');
 
